dubbo-monitor/dubbo-monitor-default/src/test/java/org/apache/dubbo/monitor/dubbo/DubboMonitorTest.java

```diff
@@ -103,22 +103,23 @@ public void testCount() throws Exception {
                 .addParameter(MonitorService.CONCURRENT, 1)
                 .addParameter(MonitorService.MAX_CONCURRENT, 1);
         monitor.collect(statistics);
+        monitor.send();
         while (lastStatistics == null) {
             Thread.sleep(10);
         }
-        Assertions.assertEquals(lastStatistics.getParameter(MonitorService.APPLICATION), "morgan");
-        Assertions.assertEquals(lastStatistics.getProtocol(), "dubbo");
-        Assertions.assertEquals(lastStatistics.getHost(), "10.20.153.10");
-        Assertions.assertEquals(lastStatistics.getParameter(MonitorService.APPLICATION), "morgan");
-        Assertions.assertEquals(lastStatistics.getParameter(MonitorService.INTERFACE), "MemberService");
-        Assertions.assertEquals(lastStatistics.getParameter(MonitorService.METHOD), "findPerson");
-        Assertions.assertEquals(lastStatistics.getParameter(MonitorService.CONSUMER), "10.20.153.11");
-        Assertions.assertEquals(lastStatistics.getParameter(MonitorService.SUCCESS), "1");
-        Assertions.assertEquals(lastStatistics.getParameter(MonitorService.FAILURE), "0");
-        Assertions.assertEquals(lastStatistics.getParameter(MonitorService.ELAPSED), "3");
-        Assertions.assertEquals(lastStatistics.getParameter(MonitorService.MAX_ELAPSED), "3");
-        Assertions.assertEquals(lastStatistics.getParameter(MonitorService.CONCURRENT), "1");
-        Assertions.assertEquals(lastStatistics.getParameter(MonitorService.MAX_CONCURRENT), "1");
+        Assertions.assertEquals("morgan", lastStatistics.getParameter(MonitorService.APPLICATION));

```

Comment:
What is the role of swapping positions here?

Reply:
Expected value should be the first parameter of `asertEquals`.
for example, this is current error message which `expected` value and `actual` value was swapped:
```
[[1;31mERROR[m] testCount  Time elapsed: 0.075 s  <<< FAILURE!
org.opentest4j.AssertionFailedError: expected: <0> but was: <1>
	at org.apache.dubbo.monitor.dubbo.DubboMonitorTest.testCount(DubboMonitorTest.java:116)
```
it should be:
"expected: <1> but was: <0>"